packages/react-dom/src/events/__tests__/EnterLeaveEventPlugin-test.js

```diff
@@ -12,27 +12,28 @@
 var React;
 var ReactDOM;
 
-const simulateMouseEvent = (node, type, relatedTarget) => {
-  node.dispatchEvent(
-    new MouseEvent(type, {
-      bubbles: true,
-      cancelable: true,
-      relatedTarget,
-    }),
-  );
-};
-
 describe('EnterLeaveEventPlugin', () => {
+  var container;
+
   beforeEach(() => {
     jest.resetModules();
 
     React = require('react');
     ReactDOM = require('react-dom');
+
+    // The container has to be attached for events to fire.
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  afterEach(() => {
+    document.body.removeChild(container);
```

Comment:
Note: unlike in existing tests, this actually ensures it's removed even if the test fails.